include/openssl/x509.h

```diff
@@ -517,6 +518,9 @@ EC_KEY *d2i_EC_PUBKEY(EC_KEY **a, const unsigned char **pp, long length);
 DECLARE_ASN1_FUNCTIONS(X509_SIG)
 void X509_SIG_get0(X509_ALGOR **palg, ASN1_OCTET_STRING **pdigest,
                    X509_SIG *sig);
+void X509_SIG_get0_const(const X509_ALGOR **palg,
```

Comment:
Is `X509_SIG_get0` a new API in 1.1.0? If so, can we
1) make `X509_SIG` the first argument;
2) make the const version simply `get0`, and the mutable one, if it's needed, `get0_mutable`? I.e. const-get0 is the default (from now on at least).